.circleci/config.yml

```diff
@@ -61,7 +61,9 @@ jobs:
 
     steps:
       - checkout
+      - base-tools/install
```

Comment:
Are all these diffs intentional--does --strict require Python stuff? Just making sure this wasn't a merge conflict or something.

Reply:
Yes, they are intentional. I needed the install base-tools to get Python for the "build" workflow, which previously only built things with node. A lot of the other changed lines are just VSCode removing trailing whitespace.